website/docs/language/settings/backends/configuration.html.md

```diff
@@ -5,11 +5,17 @@ page_title: "Backend Configuration - Configuration Language"
 
 # Backend Configuration
 
+Each Terraform configuration can specify a backend, which defines where
+[state](/docs/language/state/index.html) snapshots are stored.
 
-Each Terraform configuration can specify a backend, which defines exactly where
-and how operations are performed, where [state](/docs/language/state/index.html)
-snapshots are stored, etc. Most non-trivial Terraform configurations configure
-a remote backend so that multiple people can work with the same infrastructure.
+Backend configuration is not needed when using Terraform Cloud, because
+Terraform Cloud automatically manages state in the workspaces your configuration
+is associated with. If your configuration includes
+[a `cloud` block](/docs/language/settings/terraform-cloud.html),
+it cannot include a `backend` block.
```

Comment:
```suggestion
You do not need to configure a backend when using Terraform Cloud because
Terraform Cloud automatically manages state in the workspaces associated with your configuration. If your configuration includes [a `cloud` block](/docs/language/settings/terraform-cloud.html), it cannot include a `backend` block.
```